src/js/services/getImgService.js

```diff
@@ -0,0 +1,46 @@
+(function (angular) {
+    'use strict';
```

Comment:
Can this line be moved outside of the function now, similar to `flickr-module.js`?